woocommerce/woocommerce-android
# Feature/7510 account creation UI

Closes: #7510


### Description
Adds UI to the first step of the store creation flow: Account creation. 
This PR also includes adding Android Navigation starting from `LoginActivity` so we can use `nav_graphs` from now on in the store creation flow. I presume login simplification will also follow this road when decoupling from the authentication library. 

This also adds a feature flag to only display the store creation access point for debug builds. Please don't pay much attention to the newly added "Get started" button in `LoginPrologue` as the design is not final. 

### Testing instructions
- Open the app and navigate to the new screen for account creation
- Check that "Accept terms of Service" opens the correct website
- Check that back navigation works as expected

### Images/gif


https://user-images.githubusercontent.com/2663464/195661508-a6ab5d63-bb5c-4c39-88e2-2b72ca129f46.mp4
